runtime/shared_common/shrinit.cpp

```diff
@@ -1471,10 +1468,6 @@ hookFindSharedClass(J9HookInterface** hookInterface, UDATA eventNum, void* voidD
 		testForBytecodeModification(vm);
 	}
 
-#ifdef LINUXPPC
-	compilerBugWorkaround = localRuntimeFlags & J9SHR_RUNTIMEFLAG_CACHE_INITIALIZATION_COMPLETE;
-#endif
-
 	if (!(localRuntimeFlags & J9SHR_RUNTIMEFLAG_CACHE_INITIALIZATION_COMPLETE) ||
```

Comment:
I don't think we have information on which compiler version was affected? I'd be more comfortable with this change if the checks here are re-written to use OMR_ARE_ANY_BITS_SET(), i.e. !OMR_ARE_ANY_BITS_SET(localRuntimeFlags, J9SHR_RUNTIMEFLAG_CACHE_INITIALIZATION_COMPLETE) etc.

Reply:
do you have any preference on OMR_ARE_ANY_BITS_SET vs J9_ARE_ANY_BITS_SET ? I'm inclined to use the J9 version as it is used elsewhere in the file

Reply:
Sure, J9_ARE_ANY_BITS_SET is good.